client/src/helpers/route.ts

```diff
@@ -7,20 +7,23 @@ export class Routes {
   static MainSearch = `${ROOT}search`;
   static Patient = `${ROOT}${ROUTE_PATIENT}`;
   static PatientSearch = `${ROOT}${ROUTE_PATIENT}/${ROUTE_PATIENT_SEARCH}`;
-  static PatientSearchArranger = `${Routes.PatientSearch}/graphql`;
+  static PatientSearchOld = `${Routes.PatientSearch}/old`;
   static PatientVariants = `${ROOT}${ROUTE_PATIENT}/:uid/variant`;
   static Root = `${ROOT}`;
   static Submission = `${ROOT}submission`;
   static Variant = `${ROOT}variantDetails`;
 
   public static getPatientPath(uid?: string, tab?: string): string {
-    return uid && tab ?
-    `${Routes.Patient}/${uid}/#${tab}` :
-    uid ? `${Routes.Patient}/${uid}` : `${Routes.Patient}/:uid`
+    return uid && tab
```

Comment:
(optional): I personally think it d be more readable to use if/else here rather than chained ternaries